grails-app/views/admin/editProject.gsp

```diff
@@ -11,33 +11,35 @@
         <g:form action="saveProject" class="form-horizontal">
             <g:hiddenField name="projectId" value="${projectInstance?.id}" />
 
-            <div class="control-group">
-                <label class="control-label" for="name">Name</label>
-                <div class="controls">
-                    <g:textField name="name" value="${projectInstance?.name}"/>
+            <form class="form-horizontal">
```

Comment:
I think this tag is redundant, the <g:form> tag above will provide the form element with the url set to the controller, action, etc params and with class form-horizontal